build/lib/gradio/embeddings.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+
+SMALL_CONST = 1e-10
+
+def embed_text(text):
```

Comment:
why is embed_text not directly inside the Textbox component? no need to be in a separate file